tests/integration/kubernetes/gha-run.sh

```diff
@@ -241,6 +241,16 @@ install_kata_tools_placeholder() {
 		"after CI picks up the gha yaml changes required to test that installation."
 }
 
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
```

Comment:
Please, Use `${SNAPSHOTTER:-}` here, just to make sure that it won't break in workflows where the snapshotter is not set.

Reply:
I'd also add a TODO comment/echo.

Reply:
> Please, Use ${SNAPSHOTTER:-} here, just to make sure that it won't break in workflows where the snapshotter is not set.

Fixed.

Reply:
@fidencio , @ChengyuZhu6 I'm flagging this thread as solved to facilitate on our eyes during the review. Please feel free to reopen if need.